src/NovelAudioService.cpp

```diff
@@ -7,28 +7,26 @@
 
 namespace NovelRT {
 
-NovelAudioService::NovelAudioService() : _nextChannel(1), _musicTime(0), _musicPausedTime(0), isInitialized(false) {
+NovelAudioService::NovelAudioService() : _nextChannel(1), _musicTime(0), _musicPausedTime(0), isInitialized(false), _console(NovelUtilities::CONSOLE_LOG_AUDIO) {
   initializeAudio();
 }
 
 bool NovelAudioService::initializeAudio() {
   if (SDL_InitSubSystem(SDL_INIT_AUDIO) < NovelUtilities::SDL_SUCCESS)
   {
-    std::cout << "ERROR: Cannot play audio!" << std::endl;
-    std::cout << "SDL Error: " << SDL_GetError() << std::endl;
+    _console.log("Cannot play audio: " + std::string(SDL_GetError()) , LogLevel::ERR);
   }
   else if (Mix_OpenAudio(44100, MIX_DEFAULT_FORMAT, 2, 2048) < NovelUtilities::SDL_SUCCESS)
   {
-    std::cout << "ERROR: Cannot play audio!" << std::endl;
-    std::cout << "SDL_Mixer Error: " << Mix_GetError() << std::endl;
+    _console.log("Cannot play audio: " + std::string(Mix_GetError()), LogLevel::ERR);
   }
   else if (Mix_AllocateChannels(NOVEL_MIXER_CHANNELS) < NovelUtilities::SDL_SUCCESS)
   {
-    std::cout << "ERROR: Failed to allocate channels." << std::endl;
+    _console.log("Failed to allocate channels.", LogLevel::ERR);
   }
   else
   {
-    std::cout << "INFO: Mixer engaged!" << std::endl;
+    _console.log("SDL2_Mixer Initialized.", LogLevel::INFO);
```

Comment:
nit: Maybe have helper functions like `logInfo`, `logError`, etc?

Reply:
Added - this is resolved and will be used throughout.